sources/lib/Converter/PgEntity.php

```diff
@@ -231,20 +231,19 @@ public function toPgStandardFormat($data, $type, Session $session)
 
         $fields = $this->getFields($data);
 
-        return
-            sprintf("(%s)",
-                join(',', array_map(function ($val) {
-                    if ($val === null) {
-                        return '';
-                    } elseif ($val === '') {
-                        return '""';
-                    } elseif (preg_match('/[,\s]/', $val)) {
-                        return sprintf('"%s"', str_replace('"', '""', $val));
-                    } else {
-                        return $val;
-                    }
-                }, $this->createHydrationPlan($session)->freeze($fields)
-                ))
-            );
+        return sprintf(
+            "(%s)",
+            join(',', array_map(function ($val) {
```

Comment:
there is multitude of join-s througout the code and php innspections EA extended in phpstorm is bugging me with join is an alias, please use implode instead...